pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -5,33 +5,40 @@
   front_title: 'Join the largest learning event in history, %{campaign_date}'
   coming_soon: 'Coming Soon'
 
-  campaign_date_start_short: "Dec. 6"
-  campaign_date_start_long: "December 6"
-  campaign_date_short: "Dec. 6-12"
-  campaign_date_full: "December 6-12"
-  campaign_date_year: "2021"
-  campaign_date_full_year: "December 6-12, 2021"
+  campaign_date_start_short: "Dec. 5"
+  campaign_date_start_long: "December 5"
+  campaign_date_short: "Dec. 5-11"
+  campaign_date_full: "December 5-11"
+  campaign_date_year: "2022"
+  campaign_date_full_year: "December 5-11, 2022"
+
+  nonus_campaign_date_start_short: "Oct. 1"
+  nonus_campaign_date_start_long: "October 1"
+  nonus_campaign_date_short: "Oct. 1 - Dec. 18"
+  nonus_campaign_date_full: "October 1 - December 18"
+  nonus_campaign_date_year: "2022"
+  nonus_campaign_date_full_year: "October 1, 2022 - December 18, 2022"
 
   latam_campaign_date_start_short: "Oct. 1"
   latam_campaign_date_start_long: "October 1"
-  latam_campaign_date_short: "Oct. 1 - Dec. 12"
-  latam_campaign_date_full: "October 1 - December 12"
-  latam_campaign_date_year: "2021"
-  latam_campaign_date_full_year: "October 1, 2021 - December 12, 2021"
+  latam_campaign_date_short: "Oct. 1 - Dec. 18"
```

Comment:
Do we need to keep these constants around if everyone is using `nonus`?

Reply:
I didn't want to lose the translation history on these strings, and thought we might go back to using them at some point, so figured it was worth keeping them around for now...